pkg/network/events/monitor.go

```diff
@@ -118,7 +116,7 @@ func (e *eventMonitor) UnregisterHandler(handler ProcessEventHandler) {
 	e.Lock()
 	defer e.Unlock()
 
-	if idx := slices.Index(e.handlers, handler); idx >= 0 {
-		e.handlers = slices.Delete(e.handlers, idx, idx+1)
-	}
+	//if idx := slices.Index(e.handlers, handler); idx >= 0 {
```

Comment:
Should be removed?

Reply:
Oh yep, thanks didn't mean to commit that - was having compile errors on my machine